core/server/api/v2/webhooks.js

```diff
@@ -0,0 +1,47 @@
+const models = require('../../models');
+const common = require('../../lib/common');
+
+module.exports = {
+    docName: 'webhooks',
+    add: {
+        statusCode: 201,
+        headers: {},
+        options: [
+            'include'
```

Comment:
What is the `include` here for?

Reply:
It was default in v0.1 We just copied it for now.

Reply:
e.g. you can say POST /recource/?include and you will receive the relations on creation.

Reply:
Ah right, what relations does it have?

Reply:
0

Reply:
😁

Reply:
So can be removed

Reply:
:stuck_out_tongue_closed_eyes:

Reply:
:D Removed 🗡